src/main/scala/name.lakhin.eliah.projects/papacarlo/lexis/Contextualizer.scala

```diff
@@ -124,7 +125,7 @@ final class Contextualizer {
     var context = entryContext
     for (token <- tokens) {
       val next = stateMachine.get(token.kind).flatMap(_.get(context.kind))
-        .getOrElse(Pair(RegularSeam, 0))
+        .getOrElse(Tuple2(RegularSeam, 0))
```

Comment:
```suggestion
        .getOrElse((RegularSeam, 0))
```

Reply:
i would keep `Tuple2(Some, Value)` for clarity
also for consistency: other code-parts use `Tuple4`
also, i guess, `Tuple2[SomeType]` is still needed